src/h5web/visualizations/shared/AxisSystem.module.css

```diff
@@ -34,7 +39,7 @@
 }
 
 .tick > text {
-  fill: currentColor;
+  fill: var(--secondary-dark);
```

Comment:
I have to say I preferred the labels and the grid black/grey. 😅 I think it's too much green now, and I feel like the green grid would not come out well with some colour maps. A very transparent black is safer.

Reply:
You could try with some greyish labels too, but be mindful that #777 is the lightest you can go for accessibility reasons.

Reply:
FYI, our colours don't have sufficient colour contrast with white (as text or bg), only with black. Here is the contrast checks for the full palette for reference: https://abc.useallfive.com/?colors[]=FFFFFF,020402,1B998B,40C9A2,B3E9DA,D9F4EC,ECFAF6,9AAE5D,C0DA74,D4E09B

Reply:
Labels are now in darkslategrey and grid/ticks in grey